features/create_event.feature

```diff
@@ -3,3 +3,42 @@ Feature: Create Event
   As a establishment owner
   I want to create an event along with its name, the band list, state, date and description and I want the
     establishment set as the owner.
+
+  Background: There are registered user, an establishment created by one of them, and there is also a band created by one of them.
+    Given There is an establishment
+
+    Scenario: 
+    Given There are Establishments
+      | user        | password | name     | address     | mail                 | mobile    |
+      | Tremola     | patata   | Tremola  | C.Major n 9 | tremola@gmail.com    | 100000001 |
+    And There is a band
+      | user          | password | web_link                             | playlist                                    | mail                 | mobile    | name      |
+      | Skeewiff      | patata   | https://soundcloud.com/skeewiff      |  https://soundcloud.com/dj-elye/sets/jazz   | skeewiff@gmail.com   | 100000001 | skeewiff  |
+    When I create a event
+      |  name            |  band                 |  state  |  date  |  description  |  establishment  |
```

Comment:
Change establishment for user, and put the username in the contents, as the event was changed